java/arcs/core/storage/ReferenceModeStore.kt

```diff
@@ -191,17 +177,10 @@ class ReferenceModeStore private constructor(
 
     override fun off(callbackToken: Int) {
         callbacks.unregister(callbackToken)
-        if (callbacks.isEmpty() && !callbacksStateChannel.isClosedForSend) {
-            try {
-                callbacksStateChannel.offer(false)
-            } catch (e: ClosedSendChannelException) {
-                // No-op. If the channel is closed (which can happen between the if's check and the
-                // offer call above), then it's no big deal.
-                log.debug {
-                    "Attempted to send false to the callbacksStateChannel when it was already " +
-                        "closed."
-                }
-            }
+        runBlocking {
```

Comment:
I think runBlocking is only allowed on JVM Kotlin so far...

Reply:
I was going to ask about this. I was using this as an alternative to changing the `off` method to be suspending (as I did in that other draft). I could do that again here.